source/develop/release-management/features/network/copy-host-networking.html.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Copy Host Networking
+category: feature
+authors: Ales Musil
+feature_name: Copy Host Networking
+feature_modules: copy-host-networking
+feature_status: In Development
+---
+
+# Copy Host Networking
+
+## Summary
+
+As it turned out setting up multiple hosts at once can be rather tedious via
```

Comment:
It is not just "at once". I'd day

Adding multiple hosts to oVirt via the UI can be rather tedious.

Reply:
Done